api/server.js

```diff
@@ -108,6 +108,17 @@ const createBareServer = function () {
     },
   };
 
+  // Force https on non-dev environments
+  if (config.environment !== 'development') {
+    serverConfiguration.routes.security = {
+      hsts: {
+        includeSubDomains: true,
+        preload: true,
```

Comment:
```suggestion
        preload: false,
```

On est pas OK pour le preload chez Pix, et de ma comprehension c'est pas le souhait d'y aller (a confirmer avec  @romain-pix-cyber eventuellement) ; en tout cas il ne faut pas le mettre a true pour l'instant

A bien enlever partout car le site sera completement innaccessible sous Chrome sinon

Reply:
AFAIK le preload tout seul ne va rien changer, et l'ajout aux listes de preload en l'état ne cassera rien en l'état chez Pix. Pas sûr d'aller tout de suite vers l'ajout sur les listes mais autant le positionner dès maintenant